infra/cifuzz/fuzz_target.py

```diff
@@ -140,6 +144,25 @@ def fuzz(self):
       return FuzzResult(testcase, stderr)
     return FuzzResult(None, None)
 
+  def delete_to_save_disk_space(self):
+    """Deletes things that are no longer needed from fuzzing this fuzz target to
+    save disk space if needed."""
+    if not self.config.low_disk_space:
+      return
+    logging.info(
+        'Deleting corpus, seed corpus and fuzz target of %s to save disk.',
+        self.target_name)
+
+    # Delete the seed corpus, corpus, and fuzz target.
+    if self.latest_corpus_path:
```

Comment:
Do we need to check os.path.exists as well.

Reply:
Done.